src/retrace/config.py.in

```diff
@@ -91,7 +91,7 @@ class Config:
             "BugzillaTriggerWords": "",
             "BugzillaRegExes": "",
             "BugzillaQueryLastChangeDelta": 168,
-            "ProcessCommunicateTimeout": 3600,
+            "ProcessCommunicateTimeout": 43200,
```

Comment:
You will need to update this value in `src/config/retrace-server.conf` as well. Also, please, add a comment there remarking that the default is equal to 12 hours. It always bugs me when I have to do arithmetic in my head :smile:

Reply:
Yes, sure will!